### ISSUE TITLE
Playing with 48 hedgehogs and Per Hedgehog Ammo is not possible

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#108]
```
What steps will reproduce the problem?
1. Select game mode with Per Hedgehog Ammo.
2. Add 6 team with 8 player each.
3. Run fight.

What is the expected output? What do you see instead?
The fight don't run, I get a error window:
"Last two engine messages:
Establishing IPC connection... ok
Ammo stores overflow"

What version of the product are you using? On what operating system?
0.9.14.1 On Windows XP SP2

Please provide any additional information below.


```

-----
Original issue reported on code.google.com by `adibiaz...@gmail.com` on 21 Nov 2010 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
As I checked, playing with more than 42 hedgehogs is not possible with Per 
Hedgehog Ammo.
```

-----
Original comment by `adibiaz...@gmail.com` on 21 Nov 2010 at 3:14

**Comment by GoogleCodeExporter** [comment]
```
Oops :D
Fixed in 4387:beb4de0af990

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Nov 2010 at 6:19
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified, nothing wrong happens when plaing wtih 48 hogs and per hedge ammo.
```

-----
Original comment by `pilarski...@gmail.com` on 29 Nov 2011 at 5:59
* Changed state: **Verified**
